tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Utils/Database.cs

```diff
@@ -16,14 +16,22 @@ public async Task Reset(IConfigurationRoot config)
         {
             using IDbConnection databaseConnection = new SqlConnection(config.GetConnectionString("CatalogueUsersAdmin"));
             await databaseConnection.ExecuteAsync("ALTER ROLE db_datareader ADD MEMBER NHSD;");
+            await databaseConnection.ExecuteAsync("ALTER ROLE db_datawriter ADD MEMBER NHSD;");
             await databaseConnection.ExecuteAsync("DELETE FROM Organisations");
             await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE SecurityStamp = 'TestUser'");
+            await databaseConnection.ExecuteAsync("DELETE FROM AspNetUsers WHERE UserName IS NULL");
```

Comment:
There are now two delete methods, any reason for the second?

Reply:
This gets rid of all the created users through the POST endpoint (as they don't have the SecurityStamp set to 'TestUser')